docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json

```diff
@@ -130,8 +129,8 @@
     "variants": {
       "$id": "#/properties/variants",
       "type": "array",
-      "title": "Variants Collection",
-      "description": "The list of variants defined for this feature. A variant represents a value of a feature flag that can be a string, a number, a boolean, or even a configuration object.",
+      "title": "Variant Collection",
+      "description": "The list of variants defined for this feature. A variant represents a configuration value of a feature flag that can be a string, a number, a boolean, or even a configuration object.",
```

Comment:
NIT: no need to stress on configuration object "... or ~even~ a configuration object".

Reply:
A second thought, should it be "JSON object"? We don't support the configuration object to be anything else, do we?

Reply:
In terms of how it's defined in app config/local files, it should always be a JSON. I can change it to say that